fastlane/metadata/de-DE/privacy_url.txt

```diff
@@ -0,0 +1 @@
+https://info.ecosia.org/privacy
```

Comment:
I guess we should already use the updated one? 😅 

It is this one: https://www.ecosia.org/privacy